.github/ISSUE_TEMPLATE/bug_report.md

```
@@ -0,0 +1,38 @@
+## Actual Behavior
+What actually happened.
+
+## Environment
+- DedicatedServerMod Version: 
+- MelonLoader Version: 
+- Schedule I Version: 
+- OS: Windows 10/11 / Windows Server 2019 / Linux
+- Build Type: Mono_Server / Il2cpp_Server
+
+## Logs
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)